app/Libraries/TextFormatter.php

```diff
@@ -67,19 +68,20 @@ private function markdownParser()
         // Get an instance of the parser and the renderer
         extract($configurator->finalize());
 
-        $this->markdownParser = $parser;
+        $this->markdownParser   = $parser;
         $this->markdownRenderer = $renderer;
 
         return $this->markdownParser;
     }
 
     /**
      * Returns the Markdown renderer
+     *
      * @return mixed
      */
     private function markdownRenderer()
     {
-        if (! is_null($this->markdownRenderer)) {
+        if (null !== $this->markdownRenderer) {
```

Comment:
Why does our Rector force this one? This is bringing back Yoda comparisons. Whatever.

Reply:
I was a bit surprised too, because it's not how I would write the code, but I have no stamina today for searching how to change this.

Reply:
Fixed... it bothered me. The CodeIgniter coding standard had this rule, which is now overridden.

Reply:
Awesome. Thank you!